packages/react/src/components/Grid/Column.js

```diff
@@ -124,6 +125,13 @@ function getClassNameForBreakpoints(breakpoints) {
 
     const name = breakpointNames[i];
 
+    // If our breakpoint is a boolean, the user has specified that the column
+    // should be "auto" at this size
+    if (typeof breakpoint === 'boolean') {
```

Comment:
Wouldn't we only want to support the value `true`? I feel like `false` shouldn't create an auto-sized column